src/main/webapp/app/shared/websocket/websocket.reducer.ts

```diff
@@ -0,0 +1,73 @@
+import { connectWebSocket, subscribeWebSocket } from 'app/shared/websocket/stomp-client.service';
+import { getToken } from 'app/shared/reducers/authentication';
+
+export const ACTION_TYPES = {
+  SET_VISUAL_DATA: 'visualData/SET_VISUAL_DATA',
+  SET_VISUAL_ERROR: 'visualData/SET_VISUAL_ERROR',
+  SET_CONNACTION_STATUS: 'visualData/SET_CONNACTION_STATUS',
```

Comment:
check the spelling of SET_CONNACTION_STATUS..should be CONNECTION

Reply:
done